### DISCUSSION TITLE
How to view Supabase logs locally

### DISCUSSION THREAD

**Discussion by irohitb** [supabase/supabase#17153] [Questions]
I have an rpc function, say something like this 

```
CREATE OR REPLACE FUNCTION public.get_all_users_with_services()
RETURNS JSONB AS $$
DECLARE
  _workspace_id BIGINT := ((current_setting('request.jwt.claims'::text, TRUE))::JSON ->> 'workspace_id')::BIGINT;
  result_array JSONB;
  active_services TEXT[];
BEGIN
  -- Get all connected services from company_users
  active_services := public.get_active_services(_workspace_id);
  RAISE LOG 'Debug: Active Service = %',  active_service;
  result_array := public.get_users_info(_workspace_id, active_services, NULL);
  RETURN result_array;
END;
$$ LANGUAGE plpgsql;
```

How can I view logs raised by `RAISE LOG` in **localhost**. I can't find any option in supabase studio. 

I tried viewing in terminal using the command from this answer: https://github.com/supabase/cli/issues/271#issuecomment-1661981609 but this returned the below screenshot. 

PS: I am bit new to postgres. 

<img width="1512" alt="Screenshot 2023-09-05 at 3 43 35 AM" src="https://github.com/supabase/supabase/assets/32276134/c3af2646-b089-4986-938f-f6bd07359da7">

**Comment by Ziinc** [comment]
Postgres logs should be visible in the studio under Logs > Postgres Logs.
What is your cli version?
